### ISSUE TITLE
Not searchable code

### ISSUE THREAD

**Issue by niksingh710** [gdh1995/vimium-c#984]
using firefox
![image](https://github.com/gdh1995/vimium-c/assets/60490474/42c82569-4fff-48d4-bd6b-2b3b62c250ec)
code inside this github block is not searchable if using `/` to search

**Comment by gdh1995** [comment]
Well it's a limitation of Firefox - its webpage APIs don't allow web-extensions to search text in `textarea`s which GitHub uses to "display code"

**Comment by niksingh710** [comment]
> Well it's a limitation of Firefox - its webpage APIs don't allow web-extensions to search text in `textarea`s which GitHub uses to "display code"

firefox `ctrl+f` is able to find text in that.

**Comment by gdh1995** [comment]
because ctrl-f doesn't use Firefox's webpage API



---Original---
From: "Nikhil ***@***.***&gt;
Date: Sun, Aug 6, 2023 14:07 PM
To: ***@***.***&gt;;
Cc: "Dahan ***@***.******@***.***&gt;;
Subject: Re: [gdh1995/vimium-c] Not searchable code (Issue #984)




  
Well it's a limitation of Firefox - its webpage APIs don't allow web-extensions to search text in textareas which GitHub uses to "display code"
  
firefox ctrl+f is able to find text in that.

**Comment by niksingh710** [comment]
wouldn't it be possible to use that instead of firefox's webpage API?

**Comment by gdh1995** [comment]
No. it is conflicted with many features of Vimium C's FindMode, including regexp-finding.





---Original---
From: "Nikhil ***@***.***&gt;
Date: Mon, Aug 7, 2023 14:29 PM
To: ***@***.***&gt;;
Cc: "Dahan ***@***.******@***.***&gt;;
Subject: Re: [gdh1995/vimium-c] Not searchable code (Issue #984)




 
wouldn't it be possible to use that instead of firefox's webpage API?

**Comment by niksingh710** [comment]
I guess then it can only be resolved by intervention of Mozilla itself.
[Issue](https://bugzilla.mozilla.org/show_bug.cgi?id=1847490) on Bugzilla.

**Comment by niksingh710** [comment]
@gdh1995 
[Issue](https://bugzilla.mozilla.org/show_bug.cgi?id=1847490)

Please check this out once.

**Comment by gdh1995** [comment]
I know the API of `browser.find.*` but it's conflicted with many features of Vimium C's FindMode, including regexp-finding. So Vimium C won't use it.
